packages/trendmicro/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.0.1"
```

```suggestion
- version: "0.1.0"
```
since this has system tests we can label it as a beta.